DebuggerFeaturePkg/DebuggerFeaturePkg.dsc

```diff
@@ -18,6 +18,8 @@
   BUILD_TARGETS                  = DEBUG|RELEASE|NOOPT
   SKUID_IDENTIFIER               = DEFAULT
 
+!include MdePkg/MdeLibs.dsc.inc

```

Comment:
I think we don't want to encourage use of MdeLibs.dsc.inc and should directly add the two stack check related libs here